x-pack/plugins/maps/migrations.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { extractReferences } from './common/migrations/references';
+
+export const migrations = {
+  'map': {
+    '7.1.0': (doc) => {
```

Comment:
do we need to maintain this every minor release? Or is this the oldest version the saved object was introduced?

Reply:
This is the version that the migration was introduced and will not need to be maintained for every minor release